### DISCUSSION TITLE
I can't find a windows exe file after building my tauri app

### DISCUSSION THREAD

**Discussion by felixfavour** [tauri-apps/tauri#9884] [Q&A] [answered]
Hi everyone, my Tauri app works just fine on development, and it bundles Mac installation apps perfectly, but I can't find any .exe files in the release folder for Windows apps.

I have very little experience with Tauri and desktop apps in general, so I may be doing something wrong. Please help or point me to any information you believe could help. Thanks.

**Comment by FabianLars** [comment]
It should be in `target/release/` if you run `tauri build` and in `target/debug/` in if you run `tauri build --debug` or `tauri dev`. Note however that this .exe is typically not shippable as-is and instead tauri expects you to ship the nsis or msi installer (in `target/debug/bundle/`).

If this doesn't help, do these paths exist? If not, then maybe there's a `.cargo/config.toml` somewhere that's modifying the target dir? If you run `tauri build` it should tell you where the files are.

**Reply by felixfavour** [reply]
This is what my target folder looks like, no .exe, just dmg files.
I should mention my development machine is a mac, does that affect anything?

<img width="208" alt="image" src="https://github.com/tauri-apps/tauri/assets/46944485/6b907d20-0305-4e4f-a917-65abc556be8a">

**Reply by FabianLars** [reply, marked as answer]
Ohhh, i misunderstood the question then. Yeah, there's no support for cross-compilation (if we ignore https://tauri.app/v1/guides/building/cross-platform/#experimental-build-windows-apps-on-linux-and-macos ) so you'll only get the bundles for the current platform.

**Reply by felixfavour** [reply]
Makes sense, thank you.

**Comment by lloydfff** [comment]
hello guys, i have been working on a small project recently, a command that could link three software and do specific action that would require hundred folds the time it now takes in the automated program, but even if i can use it directly in .bat form i can't share it with others people, that's why i have been working on the interface ui in js and have made up all the necessary files for the tauri project, but after spending hours trying to compile it to exe, i still can't, here is what i get :
<img width="1710" height="260" alt="Capture d'écran 2025-07-24 184132" src="https://github.com/user-attachments/assets/acebd277-67b2-41c7-865e-699248ef22cc" />
i hope you can help me !

**Reply by FabianLars** [reply]
try looking at the `cargo tauri info` output. This sounds like version incompatibilities (tauri v1 and v2 used together)

edit: also this is really off-topic for this discussion thread.
